### ISSUE TITLE
Master vs slave

### ISSUE THREAD

**Issue by dkjswart** [GMTrevis/Homeassistant-NodeRed-Systemair-VTR300#4]
What should the host and slave IP be? Is the host the IP of the IAM module?
Are there any of these that shouldn't be changed? If so, which of these?

**Comment by GMTrevis** [comment]
> What should the host and slave IP be? Is the host the IP of the IAM module? Are there any of these that shouldn't be changed? If so, which of these?

Yes, in Node Red should the "Host" in the "Modbus client node" and the "slave_IP" in the function nodes correspond with the IP address of your IAM module:
![IP_2](https://user-images.githubusercontent.com/69414078/170054761-7d1afda7-c2b1-489a-b974-41234a2588f1.png)
![IP_3](https://user-images.githubusercontent.com/69414078/170054764-ad6aaf04-bb4f-4188-a9b9-aeda777f35f3.png)
![IP_1](https://user-images.githubusercontent.com/69414078/170054765-beacdfcc-8ecc-4098-b8e6-25162a63e996.png)
